### PR TITLE
Docs `v3.0.0`

### PR DESCRIPTION
**Pull request by bragaz**
This PR update to docs to match Desmos `v3.0.0`. The following changes have been applied:
- Updated Profiles transactions and queries sections;
- Splitted Relationships docs from Profiles;
- Updated Relationships docs (txs, queries, types);
- Added Subspaces docs.

### CODE REVIEW

**Comment by bragaz** [comment]
@RiccardoM @dadamu Once you've approved this PR, I will run the command to created the versioned docs `v3.0.0` and merge the PR.

File: docs/02-developers/02-types/subspaces/subspace.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The `Treasury` represents the account associated with the subspace that should be used to connect external applications to verify it.
+It's a string representation of a Bech32 address that begins with the `desmos` human-readable part.
+
+## `Owner`
+The `Owner` is the address of the user that owns the subspace.
+It's a string representation of a Bech32 address that begins with the `desmos` human-readable part.
+
+## `Creator`
+The `Creator` identifies the user that has created the subspace.  
+It's a string representation of a Bech32 address that begins with the `desmos` human-readable part.
```
Comment (RiccardoM) [inline docs/02-developers/02-types/subspaces/subspace.md:32]:
```suggestion
The `Creator` identifies the Desmos address of user that has created the subspace.
```

File: docs/02-developers/03-transactions/subspaces/add-user-to-group.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# `MsgAddUserToGroup`
+This message allows to add a user to a user group.
+
+### Attributes
+|   Attribute   |  Type  | Description                                     | Required |
+|:-------------:|:------:|:------------------------------------------------|:---------|
+| `subspace_id` | Uint64 | Id of the subspace where the group exists       | yes      |
+|  `group_id`   | Uint32 | Id of the group where to add the user           | yes      |
+|     `user`     | String | Address of the user to be inserted in the group | yes      |
+|   `signer`    | String | Address of the user that sign the msg           | yes      |
```
Comment (RiccardoM) [inline docs/02-developers/03-transactions/subspaces/add-user-to-group.md:17]:
```suggestion
|   `signer`    | String | Address of the owner that is adding the user to the group  | yes      |
```

File: docs/02-developers/03-transactions/subspaces/set-user-group-permissions.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+# `MsgSetUserGroupPermissions`
+This message allows to set a user group permissions.
+
+### Attributes
+|   Attribute   |  Type  | Description                                                             | Required |
+|:-------------:|:------:|:------------------------------------------------------------------------|:---------|
+| `subspace_id` | Uint64 | Id of the subspace where the group exists                               | yes      |
+|  `group_id`   | Uint32 | Id of the group to which we need to set the permission for              | yes      |
+| `permissions` | Uint32 | Permissions to be set for the group combined and represented as integer | yes      |
+|    `signer`    | String | Address of the signer of the msg                                        | yes      |
```
Comment (RiccardoM) [inline docs/02-developers/03-transactions/subspaces/set-user-group-permissions.md:17]:
```suggestion
|    `signer`    | String | Address of the signer of the message                                        | yes      |
```

File: docs/02-developers/03-transactions/subspaces/set-user-permissions.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+# `MsgSetUserPermissions`
+This message allows setting user permissions.
+
+### Attributes
+| Attribute | Type | Description                                   | Required |
+| :-------: | :----: |:----------------------------------------------| :------- |
+| `subspace_id` | Uint64 | Id of the subspace where the group exists     | yes      |
+|  `group_id`   | Uint32 | Id of the group where to add the user         | yes      |
+| `permissions` | Uint32 | Permissions to be set for a user of the group | yes      |
+|   `signer`    | String | Address of the user that sign the msg         | yes      |
```
Comment (RiccardoM) [inline docs/02-developers/03-transactions/subspaces/set-user-permissions.md:17]:
```suggestion
|   `signer`    | String | Address of the user that sign the message         | yes      |
```

**Pull request merged**